### PR TITLE
claim ~net, ruled by Emperor Norton

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Hey mate, don't be changing my glam!

File: urb/zod/arvo/ames.hoon [hunk 1]
```diff
@@ -188,7 +188,7 @@
         0w0                           ::   76, ~wyn, Curtis Yarvin (sator)
         0w0                           ::   77, ~rec, Curtis Yarvin (sator)
         0w0                           ::   78, ~meg, Curtis Yarvin (sator)
-        0w0                           ::   79, ~net, Curtis Yarvin (sator)
+        0w0                           ::   79, ~net, ~hatteb-mitlyd (fidelus)
```
Comment (ghost) [inline urb/zod/arvo/ames.hoon:191]: This should be "fidelis", not "fidelus".

Comment (cgyarvin) [inline urb/zod/arvo/ames.hoon:191]: Rejected!  You can't be bogarting someone else's emperor, dude...

Comment (ghost) [inline urb/zod/arvo/ames.hoon:191]: Yes! I didn't think to check, sorry. Won't happen again, and thanks for the myriad corrections. Bad form, party foul.

**Pull request closed without merging**